docs/release-notes/index.rst

```diff
@@ -201,18 +201,18 @@ development keeps pace with the OS's new releases and support lifecycle.
   alternative solution, or state if no alternative exists. If necessary,
   describe the consequences of the reader's inaction. Link to relevant
   documentation, standards, or public discussion. For example, "In accordance
-  with the report, Starcraft 1.0 no longer supports encryption algorithm X. As
+  with the report, Craft Artifacts 1.0 no longer supports encryption algorithm X. As
   of this release, if you haven't already we highly recommend you immediately
   switch to encryption algorithm Y to ensure your data stays protected. For
   more details about this decision and our policy, see ```Security notice on
   encryption X <>`_`` on the Ubuntu blog.">
 
 
-  Deprecated features in Starcraft 1.0
-  ------------------------------------
+  Deprecated features in Craft Artifacts 1.0
+  ------------------------------------------
 
-  The following features should no longer be used in Starcraft 1.0, and will be
-  removed in Starcraft 1.1.
+  The following features should no longer be used in Craft Artifacts 1.0, and will be
+  removed in Craft Artifacts 1.1.
```

Comment:
not related to your PR, but this seems aggressive, the time to warn would have been during the previous release and the time to remove the major release

Reply:
That's a great point. How about:

```rst
Upcoming deprecations as of Craft Artifacts 1.0
-----------------------------------------------

The following features should no longer be used as of Craft Artifacts 1.1, and will be
removed in Craft Artifacts 2.0.
```